core/internal/testutils/testutils.go

```diff
@@ -391,3 +391,8 @@ func AssertCount(t *testing.T, db *sqlx.DB, tableName string, expected int64) {
 func NewTestFlagSet() *flag.FlagSet {
 	return flag.NewFlagSet("test", flag.PanicOnError)
 }
+
+// Ptr takes pointer of anything
+func Ptr[T any](v T) *T {
```

Comment:
nit/ having a local copy makes for a shorter local func `ptr` instead of `testutils.Ptr`

Reply:
Yeah but this can be used anywhere so you don't have to rewrite the function

Reply:
https://go-proverbs.github.io/
["A little copying is better than a little dependency."](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)
:shrug: